### PR TITLE
Uros/admin explore UI

### PR DESCRIPTION
**Pull request by stesevicuros**
## Description

Create Admin Explore Page UI

## Screenshots

image

image

### CODE REVIEW

File: src/components/common/top-nav-bar/TopNavBar.tsx [hunk 1]
```diff
@@ -3,13 +3,16 @@ import { useNavigate } from "react-router-dom";
...
 
-export default function TopNavBar({ title }: Props) {
+export default function TopNavBar({ title, borderBottom = true }: Props) {
 	const navigate = useNavigate();
 
 	return (
-		<div className="top-nav-bar">
+		<div
+			className="top-nav-bar"
+			style={{ borderBottom: `${borderBottom ? "#e6e6e6 1px solid" : "none"}` }}>
```
Comment (g-popovic) [inline src/components/common/top-nav-bar/TopNavBar.tsx:15]: $light-gray

File: public/images/icons/active-icon.svg
Comment (g-popovic) [inline public/images/icons/active-icon.svg:1]: Naughty naughty

**Review by g-popovic — APPROVED** [review]
Other than that looks good

File: src/components/common/top-nav-bar/TopNavBar.tsx (hunk 1 above)
Comment (stesevicuros) [reply src/components/common/top-nav-bar/TopNavBar.tsx:15]: Does not work though

**Pull request merged**